sdk/arc-scvmm/Azure.ResourceManager.ArcScVmm/src/Generated/Models/ErrorResponse.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.ArcScVmm.Models
+{
+    /// <summary> Error response. </summary>
+    internal partial class ErrorResponse
+    {
+        /// <summary> Initializes a new instance of ErrorResponse. </summary>
+        internal ErrorResponse()
+        {
+        }
+
+        /// <summary> Initializes a new instance of ErrorResponse. </summary>
+        /// <param name="error"> The error details. </param>
+        internal ErrorResponse(ErrorDefinition error)
+        {
+            Error = error;
+        }
+
+        /// <summary> The error details. </summary>
+        public ErrorDefinition Error { get; }
```

Comment:
It looks like there is a gap between the definition of `ErrorDefinition` and [ResponseError](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResponseError.cs), but I think it's better to replace this type with ResponseError, which is what we made in other RPs.